storage/PreloaderThread.cpp

```diff
@@ -49,37 +49,46 @@ void PreloaderThread::run() {
   std::size_t blocks_loaded = 0;
 
   for (const CatalogRelation &relation : database_) {
-    if (relation.hasPartitionScheme()) {
+    if (relation.hasPartitionScheme() && relation.hasNUMAPlacementScheme()) {
```

Comment:
Because in `CatalogRelation` (https://github.com/pivotalsoftware/quickstep/blob/master/catalog/CatalogRelation.hpp#L144), it is:
```
#ifdef QUICKSTEP_HAVE_LIBNUMA
  ...
  bool hasNUMAPlacementScheme() const {
    return placement_scheme_ != nullptr;
  }
  ...
#endif
```
Perhaps either move that `hasNUMAPlacementScheme()` definition out of the `#ifdef` block, or move this  `hasNUMAPlacementScheme()` call into the `#ifdef` block.